rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -83,6 +82,31 @@ def run_service(self):
             endpoint=self.execute_graph_block,  # type: ignore
             methods=["POST"],
         )
+        router.add_api_route(
+            path="/templates",
+            endpoint=self.get_templates,
+            methods=["GET"],
+        )
+        router.add_api_route(
+            path="/templates",
+            endpoint=self.create_new_template,
+            methods=["POST"],
+        )
+        router.add_api_route(
+            path="/templates/{graph_id}",
+            endpoint=self.update_graph,
+            methods=["PUT"],
+        )
+        router.add_api_route(
+            path="/templates/{graph_id}",
+            endpoint=self.get_template,
+            methods=["GET"],
+        )
+        router.add_api_route(
+            path="/templates/{graph_id}/versions",
+            endpoint=self.get_graph_all_versions,
+            methods=["GET"],
+        )
```

Comment:
I'd like to reorder these and put them next to the `/graph` sister endpoint to make them easier to find in the list